packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -759,4 +759,16 @@ export default class CallDiagnosticMetrics extends StatelessWebexPlugin {
     // @ts-ignore
     return this.webex.prepareFetchOptions(request);
   }
+
+  /**
+   * Returns true if the specified serviceErrorCode maps to an expected error.
+   * @param {number} serviceErrorCode the service error code
+   * @returns {boolean}
+   */
+  public isServiceErrorExpected(serviceErrorCode: number): boolean {
+    const clientErrorCode = SERVICE_ERROR_CODES_TO_CLIENT_ERROR_CODES_MAP[serviceErrorCode];
+    const clientErrorPayload = CLIENT_ERROR_CODE_TO_ERROR_PAYLOAD[clientErrorCode];
+
+    return clientErrorPayload?.category === 'expected';
```

Comment:
Should this (and other instances of 'expected') be constants?  Seems like they would be in cantina, but not sure about sdk conventions.

Reply:
I would prefer it, but that's not the convention in the sdk.